scripts/popup.js

```diff
@@ -23,7 +23,42 @@ function view_all_function(){
 	document.location.href = open_url;
 }
 
+function facebook_function(){
+	var link = "https://www.facebook.com/sharer/sharer.php?u=";
+	chrome.tabs.getSelected(null,function(tab) {
+		chrome.windows.create({'url': link+tab.url, 'type': 'popup', 'width' : 500, 'height' : 400}, function(window) {
+		});
+	});
+}
+
+function google_function(){
+	var link = "https://plus.google.com/share?url=";
+	chrome.tabs.getSelected(null,function(tab) {
+		chrome.windows.create({'url': link+tab.url, 'type': 'popup', 'width' : 500, 'height' : 400}, function(window) {
+		});
```

Comment:
@rgpt thanks for taking the time to go through my code and reviewing it. As for your review, I didn't understand what you want to convey with "share the search on Wayback's page"? It would be better if you elaborate your query.